.github/workflows/ci-tests.yml

```diff
@@ -25,6 +32,13 @@ jobs:
           - ubuntu-22.04
           - macos-12
           - windows-2022
+        run-all: 
+          - ${{ inputs.test-macos-and-windows == true || github.ref == 'refs/heads/main' }}
```

Comment:
If this actually works, it is indeed cleaner and I'd like to see it in monorepo.
I was trying to conditionally skip the job, but `input` wasn't available there.

Reply:
Yes, please :pray: